core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/migration/Shared.scala

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.dsde.rawls.monitor.migration
+
+import cats.implicits.{catsSyntaxOptionId, toTraverseOps}
+import org.broadinstitute.dsde.rawls.RawlsException
+
+object Shared {
```

Comment:
rename to something more informative/specific. Maybe `MigrationHelpers`?

Reply:
Sure